### ISSUE TITLE
Ranger crashes when running :trash

### ISSUE THREAD

**Issue by retchohrips** [ranger/ranger#2897]
#### Runtime Environment

- Operating system and version: Arch (linux 6.5.2.arch1-1)
- Terminal emulator and version: Kitty 0.29.2
- Python version: 3.11.5
- Ranger version/commit: 1.9.3
- Locale: en_US.UTA-8

#### Current Behavior
Ranger crashes upon running `:trash`.

#### Expected Behavior
Send file to trash.

#### Context

I haven't changed any settings, fresh Ranger install. Tried to trash a folder in my home directory and Ranger crashed.

#### Steps to reproduce
1. Run `:trash`

#### Traceback

```
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/ranger/core/main.py", line 201, in main
    fm.loop()
  File "/usr/lib/python3.11/site-packages/ranger/core/fm.py", line 383, in loop
    ui.handle_input()
  File "/usr/lib/python3.11/site-packages/ranger/gui/ui.py", line 266, in handle_input
    self.handle_key(key)
  File "/usr/lib/python3.11/site-packages/ranger/gui/ui.py", line 196, in handle_key
    elif not DisplayableContainer.press(self, key):
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/ranger/gui/displayable.py", line 275, in press
    focused_obj.press(key)
  File "/usr/lib/python3.11/site-packages/ranger/gui/widgets/console.py", line 188, in press
    self.type_key(key)
  File "/usr/lib/python3.11/site-packages/ranger/gui/widgets/console.py", line 214, in type_key
    self._answer_question(answer)
  File "/usr/lib/python3.11/site-packages/ranger/gui/widgets/console.py", line 197, in _answer_question
    callback(answer)
  File "/usr/lib/python3.11/site-packages/ranger/config/commands.py", line 759, in _question_callback
    self.fm.execute_file(files, label='trash')
  File "/usr/lib/python3.11/site-packages/ranger/core/actions.py", line 459, in execute_file
    filenames = [f.path for f in files]
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/ranger/core/actions.py", line 459, in <listcomp>
    filenames = [f.path for f in files]
                 ^^^^^^
AttributeError: 'str' object has no attribute 'path'

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues
```

**Comment by Souravpakhira** [comment]
Yes im also facing the same issue

**Comment by windupbird144** [comment]
This has already been fixed in 277e1db7.

> Operating system and version: Arch (linux 6.5.2.arch1-1)

extra/ranger has the bug you describe, you can install ranger-git from AUR.
